cohttp-lwt-unix/src/io.ml

```diff
@@ -29,17 +35,25 @@ type conn = Conduit_lwt_unix.flow
 let src = Logs.Src.create "cohttp.lwt.io" ~doc:"Cohttp Lwt IO module"
 module Log = (val Logs.src_log src : Logs.LOG)
 
-let with_closed_handler f ~if_closed =
+let wrap_read f ~if_closed =
   (* TODO Use [Lwt_io.is_closed] when available:
      https://github.com/ocsigen/lwt/pull/635 *)
```

Comment:
Does this TODO make sense to implement (or delete) here?  Lwt_io.closed has been available since lwt.4.2.0

Reply:
Probably, but I think that should be a separate PR.